basic/README.adoc

```diff
@@ -249,7 +249,7 @@ This section contains the barebones information to get the JavaScript bits off t
 ====
 [source,xml,indent=0]
 ----
-include::pom.xml[tag=frontend-maven-plugin]
+include::../pom.xml[tag=frontend-maven-plugin]
```

Comment:
Hi, @mtravnicek, sorry for the misunderstanding.

> i dont even see any relevant file being commited...

It's a subtle change, so I understand why you missed it. Let me see if I can shed some light on the problem.

The tutorial has snippets that refer to actual code. The full configuration for the `frontend-maven-plugin` used to live in the `basic` module's POM, but at [some point](fb8ed20001789e2c43c2e1110773586f04a11167), that configuration was pulled up to the parent POM so that it could be shared with the other modules. 

When that configuration got pulled up, it didn't break the functionality of the `basic` module, but it did break the tutorial, because those following along now can't tell what to put for configuration in the `<plugin>` element—all they get is a blank plugin configuration.

![image](https://user-images.githubusercontent.com/19827042/105391763-ae948f80-5bd7-11eb-8b3f-05eaad876e79.png)


To fix the problem, I told the tutorial to look at the parent POM (`../pom.xml`) instead of the POM in the `basic` module (`pom.xml`). It even already had the tags delineating [the plugin section](https://github.com/UgmaDevelopment/tut-react-and-spring-data-rest/blob/pom-fix/pom.xml#L23-L59).

Thanks for commenting! If you're seeing a problem I'm not, feel free to clarify further.

Reply:
@mtravnicek, when I run the basic module, from the original repo, it works as expected. Does it work for you?